model/webhook_dispatcher.py

```diff
@@ -21,5 +21,36 @@
 class WebhookDispatcher(NamedModelElement):
     def _required_attributes(self):
         return {
-
+            'image_reference',
+            'ingress_host',
+            'concourse_cfgs',
+            'job_mappings',
+            'github_cfgs',
+            'tls_config',
+            'secrets_server',
+            'kubeconfig',
         }
+
+    def image_reference(self):
+        return self.raw.get('image_reference')
+
+    def ingress_host(self):
+        return self.raw.get('ingress_host')
+
+    def concourse_cfgs(self):
+        return self.raw.get('concourse_cfgs')
+
+    def job_mappings(self):
+        return self.raw.get('job_mappings')
+
+    def github_cfgs(self):
+        return self.raw.get('github_cfgs')
+
+    def tls_config(self):
+        return self.raw.get('tls_config')
+
+    def secrets_server(self):
+        return self.raw.get('secrets_server')
+
+    def kubeconfig(self):
```

Comment:
If I properly recall, this attribute refers to a kubernetes-cfg name? if so, name it accordingly (kubernetes_cfg_name)

Reply:
Done